Dockerfile.nginx

```diff
@@ -0,0 +1,9 @@
+FROM nginx:latest
+
+ARG NGINX_HOST
+
+# Add self signed certificate to nginx image
+RUN apt-get update && apt-get install -y openssl
+RUN mkdir -p /etc/nginx/certs/self-signed/
+RUN openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/nginx/certs/self-signed/my-computer.rec.la.key -out /etc/nginx/certs/self-signed/my-computer.rec.la.crt -subj "/C=US/ST=Florida/L=Orlando/O=Development/OU=Dev/CN=my-computer.rec.la"
```

Comment:
bundle the [rec.la](https://github.com/pryv/rec-la/tree/master/src) certificates instead of self-signed ones. They don't require any tool setup.